addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -452,13 +452,21 @@ registry.slider = publicWidget.Widget.extend({
         // Initialize carousel and pause if in edit mode.
         this.$target.carousel(this.editableMode ? 'pause' : undefined);
         $(window).on('resize.slider', _.debounce(() => this._computeHeights(), 250));
-        if (this.editableMode) {
-            // Prevent carousel slide to be an history step.
-            this.$target.on('slide.bs.carousel.slider', () => {
-                this.options.wysiwyg.odooEditor.observerUnactive();
-            });
-            this.$target.on('slid.bs.carousel.slider', () => {
-                this.options.wysiwyg.odooEditor.observerActive();
+
+        if (this.editableMode && this.$target[0].matches("section > .carousel")) {
```

Comment:
`$target`is a thing of the past for publicWidget. Use `this.$el` or, here, ` this.el`.

Although, why do we have a specific behavior for carousel which are in sections? Is it useful to check? :thinking:

Reply:
> Use this.$el or, here,  this.el

Okay

> why do we have a specific behavior for carousel which are in sections

Because only the carousels matching this selector have the `Carousel` and `CarouselItem` options :slightly_smiling_face: 
These are the only snippets/options where the sliding is important to take into account.
As explained in the third commit, the other types of carousels (like the dynamic carousel or "Image Gallery") do not have a sliding that impacts the options:
- in dynamic carousels, no option needs to slide
- in "Image-Gallery", the only sliding that appears is when reordering the slides and it is not async since the `slide` class is removed from the `.carousel` element before, so it is done directly without needing the animation

And when testing, we can even deactivate the observer without breaking anything (that's what I therefore did in the third commit, but with the mutations)

At first, I was targeting all carousels, but when I went to add the click listener in these other ones, I noticed that it was pointless and therefore I just limited this to the "real" carousels (so with the options) :slightly_smiling_face:

Reply:
Ok :+1: 

> Because only the carousels matching this selector have the Carousel and CarouselItem options 🙂

Worth that as a comment on top of that `if` :point_up: